integration-cli/requirements.go

```diff
@@ -40,7 +40,7 @@ var (
 		func() bool {
 			// Set a timeout on the GET at 15s
 			var timeout = time.Duration(15 * time.Second)
-			var url = "https://hub.docker.com"
+			var url = "http://www.baidu.com"
```

Comment:
Heh, I thought that would probably be the case; given that we have quite some contributors from China; do you think we should have some alternatives?